### PR TITLE
data:LaPulgaaa

### PR DESCRIPTION
**Pull request by LaPulgaaa**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5262"></a>

### CODE REVIEW

File: data/LaPulgaaa.json
Comment (priyankarpal) [inline data/LaPulgaaa.json:5]: Currently, you don't need to add an avatar code. You can remove it.
```suggestion

```

File: data/LaPulgaaa.json
Comment (priyankarpal) [inline data/LaPulgaaa.json:13]: Change this to `GitHub`

``` suggestion
      "name": "GitHub",
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Review by priyankarpal — APPROVED** [review]
Looks good now. 🎉

**Comment by LaPulgaaa** [comment]
> Looks good now. 🎉

do i have to wait now for the pr to get merged?

**Comment by priyankarpal** [comment]
> > Looks good now. tada
> 
> do i have to wait now for the pr to get merged?

Yes wait until the maintainer merge it.

**Review by SurajPratap10 — APPROVED** [review]
LGTM

**Review by devvsakib — APPROVED** [review]

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks for your contribution! 🎉 

<p align="center"></p>

**Pull request merged**
